src/components/LeftSidebar/SearchConversationsResults/SearchConversationsResults.vue

```diff
@@ -0,0 +1,314 @@
+import Conversation from '../ConversationsList/Conversation.vue'
+
+import { ATTENDEE, CONVERSATION, AVATAR } from '../../../constants.ts'
+import { useSettingsStore } from '../../../stores/settings.js'
+import type { Conversation as TypeConversation, ParticipantSearchResult } from '../../../types/index.ts'
+
+const props = defineProps<{
+	searchText: string,
+	conversationsList: TypeConversation[],
+	contactsLoading: boolean,
+	searchResultsListedConversations: TypeConversation[],
+	searchResults: ParticipantSearchResult[],
+}>()
+
+const emit = defineEmits<{
+	(event: 'abort-search'): void,
+	(event: 'create-new-conversation', searchText: string): void,
+	(event: 'create-and-join-conversation', item: TypeConversation): void,
+}>()
+
+const isCirclesEnabled = loadState('spreed', 'circles_enabled')
+const canStartConversations = loadState('spreed', 'start_conversations')
+const settingsStore = useSettingsStore()
+const isCompact = computed(() => settingsStore.conversationsListStyle === CONVERSATION.LIST_STYLE.COMPACT)
+
+// Item's content (avatar) + internal_padding * 2 + external_padding * 2
+const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
+
+// Other sections than joined conversations in the output order
+const sections = [
+	{
+		type: 'user',
+		caption: { type: 'caption', id: 'users_caption', name: t('spreed', 'Users') },
+	},
+	{
+		type: 'group',
+		caption: { type: 'caption', id: 'groups_caption', name: t('spreed', 'Groups') },
+	},
+	{
+		type: 'circle',
+		caption: { type: 'caption', id: 'circles_caption', name: t('spreed', 'Teams') },
+	},
+	{
+		type: 'federated',
+		caption: { type: 'caption', id: 'federated_users_caption', name: t('spreed', 'Federated users') },
+	},
+] as const
+
+type SubListType = {
+	user: ParticipantSearchResult[],
+	group: ParticipantSearchResult[],
+	circle: ParticipantSearchResult[],
+	federated: ParticipantSearchResult[],
+}
+
+const searchResultsVirtual = computed(() => {
+	// Initialize
+	const virtualList = []
+
+	const lowerSearchText = props.searchText.toLowerCase()
+	const searchResultsConversationList = props.conversationsList.filter(conversation =>
+		conversation.displayName.toLowerCase().includes(lowerSearchText)
+		|| conversation.name.toLowerCase().includes(lowerSearchText)
+	)
+
+	// Add conversations section
+	virtualList.push({ type: 'caption', id: 'conversations_caption', name: t('spreed', 'Conversations') })
+	if (searchResultsConversationList.length === 0) {
+		virtualList.push({ type: 'hint', id: 'hint_conversations', hint: t('spreed', 'No matches found') })
+	} else {
+		virtualList.push(...searchResultsConversationList.map((item) => ({ type: 'conversation', id: item.id, object: item })))
+	}
+
+	// Add "New Conversation" option if allowed
+	if (canStartConversations) {
+		virtualList.push({ type: 'listItem', id: 'new_conversation', name: props.searchText, subname: t('spreed', 'New private conversation') })
```

Comment:
it's the only one in camelCase
```suggestion
		virtualList.push({ type: 'list_item', id: 'new_conversation', name: props.searchText, subname: t('spreed', 'New private conversation') })
```
'action' would also work